sdk/communication/Azure.Communication.Administration/tests/PhoneNumberAdministrationClient/PhoneNumberAdministrationClientLiveTests.cs

```diff
@@ -86,6 +86,8 @@ public async Task GetAllReservations()
             var reservations = await reservationsPagable.ToEnumerableAsync();
 
             Assert.IsNotEmpty(reservations);
+
+            await client.CancelReservationAsync(reservationOperation.Id);
```

Comment:
Is it important that we cancel reservations here? Not opposed to it but curious because users won't do that.

In the future our API should change to add `Cancel` methods on the LRO operation instead of exposing a method on the client.

Reply:
It is better that we release numbers right away as those are limited for a specific area code.